microprofile/health-smallrye/src/main/java/org/wildfly/extension/microprofile/health/deployment/CDIExtension.java

```diff
@@ -96,12 +101,21 @@ private void afterDeploymentValidation(@Observes final AfterDeploymentValidation
         addHealthChecks(HealthLiteral.INSTANCE, reporter::addHealthCheck, healthChecks);
         addHealthChecks(LivenessLiteral.INSTANCE, reporter::addLivenessCheck, livenessChecks);
         addHealthChecks(ReadinessLiteral.INSTANCE, reporter::addReadinessCheck, readinessChecks);
+        if (readinessChecks.isEmpty()) {
+            Config config = ConfigProvider.getConfig(module.getClassLoader());
+            boolean disableDefaultprocedure = config.getOptionalValue("mp.health.disable-default-procedures", Boolean.class).orElse(false);
```

Comment:
In the context of other MP specs I've seen a particular pattern of config property keys used for impl-specific config properties (something like mp.<spec>.extension.<therest> -- @pferraro knows).  Is there such a thing for health? If not is it wise to act is if there was?

Reply:
I'm not sure to understand your comment.

This property `mp.health.disable-default-procedures` is defined by the [spec itself](https://github.com/eclipse/microprofile-health/blob/2.1/spec/src/main/asciidoc/protocol-wireformat.adoc#disabling-default-vendor-procedures)

Reply:
OK; thanks.